psydac/pyccel/ast/numpyext.py

```diff
@@ -1305,19 +1305,19 @@ class Int64(NumpyInt):
 
 
 
-NumpyArrayClass = ClassDef('numpy.ndarray',
-        methods=[
-            FunctionDef('shape',[],[],body=[],
-                decorators={'property':'property', 'numpy_wrapper':Shape}),
-            FunctionDef('sum',[],[],body=[],
-                decorators={'numpy_wrapper':NumpySum}),
-            FunctionDef('min',[],[],body=[],
-                decorators={'numpy_wrapper':NumpyMin}),
-            FunctionDef('max',[],[],body=[],
-                decorators={'numpy_wrapper':NumpyMax}),
-            FunctionDef('imag',[],[],body=[],
-                decorators={'property':'property', 'numpy_wrapper':Imag}),
-            FunctionDef('real',[],[],body=[],
-                decorators={'property':'property', 'numpy_wrapper':Real}),
-            FunctionDef('diagonal',[],[],body=[],
-                decorators={'numpy_wrapper':Diag})])
+#NumpyArrayClass = ClassDef('numpy.ndarray',
+#        methods=[
+#            FunctionDef('shape',[],[],body=[],
+#                decorators={'property':'property', 'numpy_wrapper':Shape}),
+#            FunctionDef('sum',[],[],body=[],
+#                decorators={'numpy_wrapper':NumpySum}),
+#            FunctionDef('min',[],[],body=[],
+#                decorators={'numpy_wrapper':NumpyMin}),
+#            FunctionDef('max',[],[],body=[],
+#                decorators={'numpy_wrapper':NumpyMax}),
+#            FunctionDef('imag',[],[],body=[],
+#                decorators={'property':'property', 'numpy_wrapper':Imag}),
+#            FunctionDef('real',[],[],body=[],
+#                decorators={'property':'property', 'numpy_wrapper':Real}),
+#            FunctionDef('diagonal',[],[],body=[],
+#                decorators={'numpy_wrapper':Diag})])
```

Comment:
Was this dead code?

Reply:
I want to say yes. When looking for `NumpyArrayClass` in vs code I find nothing except for the definition itself. At the same time, these lines were (partly) responsible for the import issues described above, as a TypeError in the class __init__ of FunctionDef was raised multiple times.